src/client/common/experiments/service.ts

```diff
@@ -82,7 +82,7 @@ export class ExperimentService implements IExperimentService {
             this.globalState
         );
 
-        traceInfo(`Experimentation service retrieved: ${this.experimentationService}`);
+        traceVerbose(`Experimentation service retrieved: ${this.experimentationService}`);
```

Comment:
Currently not useful as we dump the entire class, hence changing to verbose (also doesn't display well in output panel, we just get `[Object object]`)